public/src/videos-page.js

```diff
@@ -0,0 +1,12 @@
+class VideosPage extends HTMLElement {
+    constructor() {
+        super();
+    }
+    connectedCallback() {
+        this.innerHTML = `Video Test`
```

Comment:
This is not previous HTML that we made on Sessions

Reply:
not sure I follow, but added the HTML in the `video test` section and did another pull request.